device-types/Cisco/FPR3110-NGFW-K9.yaml

```diff
@@ -0,0 +1,54 @@
+---
+manufacturer: Cisco
+model: FirePOWER 3110 Next-Generation Firewall
+slug: cisco-firepower-3110-next-generation-firewall
+part_number: FPR3110-NGFW-K9
+u_height: 1
+is_full_depth: true
+subdevice_role: parent
+comments: '[Cisco Firepower 3100 Series Data Sheet](https://www.cisco.com/c/en/us/products/collateral/security/firewalls/secure-firewall-3100-series-ds.html)'
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: usb
+    type: usb-a
```

Comment:
Is this a USB port for console access? If not, then please remove it from here

Reply:
```suggestion
```
No